### PR TITLE
feat(Rules): RHICOMPL-1551 add SSG popover to tab content f483843 

### PR DESCRIPTION
**Pull request by vkrizan**
Refactors the Supported SSGs link to a separate component.
Adds the popover next to the SSG version within the rules tab content.
Adds the external icon icon.
Updates the popover wording to match the behavior.

Screenshot_2021-04-07 SCAP policies - Compliance Red Hat Insights

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [X] Input Validation
- [X] Output Encoding
- [X] Authentication and Password Management
- [X] Session Management
- [X] Access Control
- [X] Cryptographic Practices
- [X] Error Handling and Logging
- [X] Data Protection
- [X] Communication Security
- [X] System Configuration
- [X] Database Security
- [X] File Management
- [X] Memory Management
- [X] General Coding Practices

### CODE REVIEW

File: src/PresentationalComponents/TabbedRules/OsVersionText.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const OsVersionText = ({ profile, newOsMinorVersion }) => (
-    `RHEL ${ profile.osMajorVersion }.${ (profile.osMinorVersion || newOsMinorVersion) }`
+    `RHEL${'\u00A0'}${ profile.osMajorVersion }.${ (profile.osMinorVersion || newOsMinorVersion) }`
```
Comment (vkrizan) [inline src/PresentationalComponents/TabbedRules/OsVersionText.js:2]: `\u00A0` --> non-breaking space

File: src/PresentationalComponents/TabbedRules/ProfileTabContent.js [hunk 2]
```diff
@@ -21,6 +22,43 @@ ProfileSystemCount.propTypes = {
...
+    </Text>
+);
+
+SSGVersionText.propTypes = {
+    profile: propTypes.object.isRequired,
+    newOsMinorVersion: propTypes.string
+};
+
+const SSGPopoverBody = ({ profile, newOsMinorVersion }) => (
+    <TextContent style={ { fontSize: 'var(--pf-c-popover--FontSize)' } }>
```
Comment (vkrizan) [inline src/PresentationalComponents/TabbedRules/ProfileTabContent.js:44]: `TextContent` uses `.pf-c-content` which overrides the font size, unfortunately.

File: src/PresentationalComponents/TabbedRules/ProfileTabContent.js [hunk 3]
```diff
@@ -21,6 +22,43 @@ ProfileSystemCount.propTypes = {
...
+        <Text>
+            This is the latest suported version of the SCAP Security Guide (SSG) for
```
Comment (sthirugn) [inline src/PresentationalComponents/TabbedRules/ProfileTabContent.js:46]: s/suported/supported

**Review by skateman — APPROVED** [review]
The Seal of Approval
Looks good, except of the typo

File: src/PresentationalComponents/TabbedRules/ProfileTabContent.js (hunk 3 above)
Comment (vkrizan) [reply src/PresentationalComponents/TabbedRules/ProfileTabContent.js]: fixed

**Review by akofink — APPROVED** [review]
Thanks, @vkrizan!

**Pull request merged**